assets/js/slider.js

```diff
@@ -0,0 +1,47 @@
+const initializeSlider = (sliderContainer, prevButton, nextButton, visibleItems) => {
+  const sliders = sliderContainer.querySelectorAll('.productSlider');
+  const productCards = sliderContainer.id==="news" ? sliderContainer.querySelectorAll('.news-card') : sliderContainer.querySelectorAll('.product-card');
```

Comment:
```suggestion
  const cards = sliderContainer.id==="news" ? sliderContainer.querySelectorAll('.news-card') : sliderContainer.querySelectorAll('.product-card');
```
Hàm này dùng chung ở nhiều chỗ nên e đặt tên biến có thể dùng chung được nha